src/app/index.ts

```diff
@@ -185,16 +190,30 @@ module.exports = class TheiaPlugin extends Base {
 
     configuring() {
         const options = this.options as any;
+        const creationDate = new Date().getFullYear().toString();
```

Comment:
should creationDate be yearCreationDate ?

Reply:
I wanted to make it more abstract. But for now I have no example where we need to use something else than year, so will change the name.